gpytorch/models/exact_gp.py

```diff
@@ -235,7 +235,7 @@ def __call__(self, *args, **kwargs):
             train_mean = full_mean.narrow(-1, 0, train_targets.size(-1))
 
             if self.prediction_strategy is None:
-                train_train_covar = full_covar[..., :num_train, :num_train].evaluate_kernel()
+                train_train_covar = full_covar[..., :num_train, :num_train]
```

Comment:
This might be dangerous perf wise. Prediction strategies cache `train_train_covar` and then use it repeatedly. Since `LazyEvaluatedKernelTensor` no longer even caches its kernel calls, won't this lead to a new kernel call every posterior variance call unless `fast_pred_var` is on?

Reply:
using cached annotation